src/quicknxs/interfaces/data_handling/instrument.py

```diff
@@ -249,13 +249,20 @@ def load_data(self, file_path, configuration=None):
             missing_keys = (polarizer > 0 and self.pol_state not in event_ws.getRun()) or (
                 analyzer > 0 and self.ana_state not in event_ws.getRun()
             )
+            # If running in unpolarized mode, no filtering is needed
+            unpolarized = polarizer == 0 and analyzer == 0
 
             if missing_keys:
                 _use_slow_flipper_log = True
                 print("\n\nMISSING POLARIZER/ANALYZER META-DATA: USING SLOW LOGS\n\n")
 
             if _use_slow_flipper_log:
                 _path_xs_list = self.dummy_filter_cross_sections(event_ws, name_prefix=temp_workspace_root_name)
+            elif unpolarized:
+                # add the expected sample log
+                event_ws.getRun()["cross_section_id"] = "Off_Off"
+                api.RenameWorkspace(str(event_ws), "%s_entry" % temp_workspace_root_name)
```

Comment:
to be on the safe side, should we clone `events_ws` instead? the other two options do not modify input `event_ws`, right?